src/routes/settings/General.svelte

```diff
@@ -150,13 +163,19 @@
   <div>
     <Toggle
       color="orange"
-      checked={$keepGamesUpdated}
+      bind:checked={$keepGamesUpdated}
       on:change={async () => {
-        $keepGamesUpdated = !$keepGamesUpdated;
-        await setAutoUpdateGames($keepGamesUpdated);
+        $uninstallOldVersions = false;
       }}
       class="mb-2">Automatically keep games updated (experimental)</Toggle
     >
+    {#if $keepGamesUpdated}
+      <Toggle
+        color="orange"
+        bind:checked={$uninstallOldVersions}
+        class="ml-14 mb-2">Automatically uninstall old versions</Toggle
```

Comment:
this should be a translatable string, so should the above `Automatically keep games updated (experimental)`